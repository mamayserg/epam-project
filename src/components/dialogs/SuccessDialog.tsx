import Dialog from "@mui/material/Dialog";
import DialogContent from "@mui/material/DialogContent";
import DialogContentText from "@mui/material/DialogContentText";
import DialogTitle from "@mui/material/DialogTitle";
import IconButton from "@mui/material/IconButton";
import CloseIcon from "@mui/icons-material/Close";
import CheckCircleIcon from '@mui/icons-material/CheckCircle';
import React from 'react';

type SuccessDialogProps = {
  open: boolean,
  handleClose: () => void
}

export default function SuccessDialog({ handleClose, open }: SuccessDialogProps) {
  
  return (
    <Dialog maxWidth={"sm"} fullWidth open={open} onClose={handleClose}>
      <IconButton
        aria-label="close"
        onClick={handleClose}
        sx={{
          position: "absolute",
          right: 8,
          top: 8,
          color: "white",
        }}
      >
        <CloseIcon />
      </IconButton>
      <DialogTitle className="mt-4 flex flex-col items-center text-white uppercase">
       <CheckCircleIcon 
        color="error"
        sx={{
          fontSize: "55px"
        }}/>
       <div className="mt-2 mb-1 title"> Congratulations !</div>
      </DialogTitle>
      <DialogContent>
        <DialogContentText className=" text-center mt-4">
         The movie has been added to database successfully
        </DialogContentText>
      </DialogContent>
    </Dialog>
  );
}
