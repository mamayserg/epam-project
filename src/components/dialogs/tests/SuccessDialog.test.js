import { render } from '@testing-library/react';
import React from "react";
import SuccessDialog from '../SuccessDialog'

describe('searchCard', ()=>{
  test('renders snapshot searchCard', () => {
   const { asFragment } = render(<SuccessDialog open={true} />);
    expect(asFragment(<SuccessDialog/>)).toMatchSnapshot();
  });
})