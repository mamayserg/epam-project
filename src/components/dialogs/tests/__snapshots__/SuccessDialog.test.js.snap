// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`searchCard renders snapshot searchCard 1`] = `<DocumentFragment />`;
